docker-compose-dev.yml

```diff
@@ -1,82 +1,48 @@
-version: "3"
+## HOW TO RUN: ##
+#
+# docker compose -f docker-compose.yml -f docker-compose-dev.yml up -d
+#
+version: "3.4"
 services:
+
   postgis:
-    image: g3wsuite/postgis:11.0-2.5
-    ports:
-      - "5439:5432"
-    environment:
-      - POSTGRES_DBNAME=${G3WSUITE_POSTGRES_DBNAME},data_testing,data_production
-      - POSTGRES_USER=${G3WSUITE_POSTGRES_USER_LOCAL}
-      - POSTGRES_PASS=${G3WSUITE_POSTGRES_PASS}
-      - ALLOW_IP_RANGE=0.0.0.0/0
-    restart: "no"
-    logging:
-      driver: "json-file"
-      options:
-        max-size: "200k"
-        max-file: "10"
+    restart: "no" # disable autorestart
     volumes:
       - g3wsuite-pg-data:/var/lib/postgresql
-    healthcheck:
-      interval: 60s
-      timeout: 30s
-      retries: 3
-      test: "pg_isready"
     networks:
       internal:
         aliases:
           - ${WEBGIS_PUBLIC_HOSTNAME}
 
-
   g3w-suite:
-    image: g3wsuite/g3w-suite:dev
     environment:
-      - G3WSUITE_TILECACHE_PATH
+      - BITBUCKET_TOKEN
+      - G3WSUITE_DEBUG=${G3WSUITE_DEBUG:-True}
+      - G3WSUITE_GUNICORN_NUM_WORKERS=${G3WSUITE_GUNICORN_NUM_WORKERS:-1}
       - G3WSUITE_QDJANGO_SERVER_URL
```

Comment:
Check if the `G3WSUITE_QDJANGO_SERVER_URL` variable is still in use nowadays, currently it doesn't seem to have any production purpose (just forgotten to delete that also inside the `docker-compose-dev.yml`?)